tests/components/light/test_init.py

```diff
@@ -128,26 +127,17 @@ async def test_services(
     )
     await hass.async_block_till_done()
 
-    ent1, ent2, ent3 = mock_light_entities
+    ent1, ent2, _ = mock_light_entities
     ent1.supported_color_modes = [light.ColorMode.HS]
-    ent3.supported_color_modes = [light.ColorMode.HS]
+    ent2.supported_color_modes = [light.ColorMode.HS]
     ent1.supported_features = light.LightEntityFeature.TRANSITION
     ent2.supported_features = (
-        light.SUPPORT_COLOR
-        | light.LightEntityFeature.EFFECT
```

Comment:
I'm not sure why we are removing `ent2` so there are no tests for effects any more?

Reply:
> I'm not sure why we are removing `ent2` so there are no tests for effects any more?

I've added it back in